### PR TITLE
OpenAPI (bac-3)

### PR DESCRIPTION
**Pull request by Mrfroxi**

### CODE REVIEW

File: src/swaggerConfig.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+let SwaggerConfig = function (app) {
+
+    const config = new DocumentBuilder()
+        .setTitle('Paim example')
+        .setDescription('The Pain API description')
+        .setVersion('1.0')
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Version should be from package.json

File: src/swaggerConfig.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        .addTag('Pain')
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Delete this tag

File: src/swaggerConfig.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+let SwaggerConfig = function (app) {
+
+    const config = new DocumentBuilder()
+        .setTitle('Paim example')
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Change title to for example "Pani service 1"

File: src/swaggerConfig.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        .setDescription('The Pain API description')
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Pani not Pain

File: src/swaggerConfig.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+let SwaggerConfig = function (app) {
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Rename function to setupSwaggerConfig or addSwaggerConfig and write it like `
function addSwaggerConfig(args) {
// some code 
}`
Add type for `app` and return type 'void'.

File: .github/workflows/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+## Changes
```
Comment (Iluhaprog) [inline .github/workflows/pull_request_template.md]: Why template placed in this PR?

File: src/swaggerConfig.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+function addSwaggerConfig(args) {
```
Comment (Iluhaprog) [inline src/swaggerConfig.ts]: Rename args to app and add types for app and return type void.

Comment (Mrfroxi) [reply src/swaggerConfig.ts]: ok

File: src/main.ts [hunk 8]
```diff
@@ -1,8 +1,13 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { addSwaggerConfig } from './swaggerConfig';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  await console.log(app);
```
Comment (Iluhaprog) [inline src/main.ts]: Delete console.log

File: .env.test [hunk 9]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_HOST=localhost
```
Comment (Iluhaprog) [inline .env.test]: Delete .env.test and .env.development and add it it .gitignore `.env*`

**Pull request merged**